src/EditorFeatures/Test/CodeFixes/CodeFixServiceTests.cs

```diff
@@ -43,7 +43,7 @@ public async Task TestGetFirstDiagnosticWithFixAsync()
 ";
         using var workspace = TestWorkspace.CreateCSharp(code, composition: s_compositionWithMockDiagnosticUpdateSourceRegistrationService, openDocuments: true);
 
-        var diagnosticService = Assert.IsType<DiagnosticAnalyzerService>(workspace.GetService<IDiagnosticAnalyzerService>());
+        var diagnosticService = workspace.GetService<IDiagnosticAnalyzerService>();
```

Comment:
removed all bizarre casting in the tests.  tests should validate the actual interface to the rest of the code.

Reply:
also, we truly only have one impl of this.  so all this type testing is really bizarre.